doc/holochain_101/src/running_an_app.md

```diff
@@ -1 +1,28 @@
-# Running Apps
+# Running An App
```

Comment:
This section does not mention networking. It could either show how to run `hc run` with env variables providing needed info for spawning an n3h instance, or at least mention that without this it only runs a mock ("local", "virtual") network..

Reply:
added networking section below

Reply:
can't find it (?)